src/mail/view/MailViewer.js

```diff
@@ -148,7 +147,7 @@ export class MailViewer {
 	mail: Mail;
 	_mailBody: ?MailBody;
 	_contrastFixNeeded: boolean;
-	_sanitizedMailBody: ?Vnode<void>; // always sanitized in this.setSanitizedMailBodyFromMail
+	_sanitizedMailBody: ?SanitizedString; // always sanitized in this.setSanitizedMailBodyFromMail
```

Comment:
Where does SanitizedString come from? Can't find it anywhere.
Also you assign to it result *after* urlify and this will of course invalidate some things, I wonder if we should switch the order even...

Reply:
was playing around with making this type-safe, for some reason flow didn't get angry when I removed the type definition...